test/onnx/gen_onnx.py

```diff
@@ -3828,6 +3828,64 @@ def instance_norm_val_3d_test():
     return ([node], [], [y], [x_tensor, scale_tensor, bias_tensor])
 
 
+@onnx_test()
+def isinf_half_test():
+    t1 = helper.make_tensor_value_info('t1', TensorProto.FLOAT16, [2, 3])
+    t2 = helper.make_tensor_value_info('t2', TensorProto.BOOL, [2, 3])
+
+    node = onnx.helper.make_node(
+        'IsInf',
+        inputs=['t1'],
+        outputs=['t2'],
+    )
+    return ([node], [t1], [t2])
+
+
+@onnx_test()
```

Comment:
I like your fp16 tests, but a couple of them could test instead for float & double -- you don't have to create additional tests, if you would like to change some of these.. 
(Optional)

Reply:
I've changed  some of them to test float and double instead.